### PR TITLE
Add GetAccountIndexes, fix create account example

### LINKED ISSUE #1478: Add a method to the account manager to get the account indexes
## Description

Add a method to the account manager to get the account indexes in the message interface

## Motivation

Firefly needs it because of some electron limitation

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Add GetAccountIndexes, fix create account example

## Links to any relevant issues

Fixes #1478 

## Type of change

- Enhancement (a non-breaking change which adds functionality)

## How the change has been tested

Running an example which calls `manager.getAccountsIndexes()`

## Change checklist

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/message_interface/message_handler.rs [hunk 1]
```diff
@@ -118,12 +118,23 @@ impl WalletMessageHandler {
         log::debug!("Message: {:?}", message);
 
         let response: Result<Response> = match message {
-            Message::CreateAccount(account) => {
-                convert_async_panics(|| async { self.create_account(&account).await }).await
+            Message::CreateAccount { alias } => {
+                convert_async_panics(|| async { self.create_account(alias).await }).await
```
Comment (thibault-martinez) [inline src/message_interface/message_handler.rs:122]: That really doesn't require a change in the bindings ?

**Review by thibault-martinez — COMMENTED** [review]
Entry in the rust CHANGELOG please

File: src/message_interface/message_handler.rs (hunk 1 above)
Comment (Thoralf-M) [reply src/message_interface/message_handler.rs:122]: No

File: CHANGELOG.md [hunk 2]
```diff
@@ -19,6 +19,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+Unreleased - YYYY-MM-DD
```
Comment (thibault-martinez) [inline CHANGELOG.md]:
```suggestion
## Unreleased - YYYY-MM-DD
```

**Review by samuel-rufi — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**